### ISSUE TITLE
Fatal error: require(): Failed opening required '/var/www/vendor/autoload.php' (include_path='.:/usr/local/lib/php') in /var/www/artisan on line 18

### ISSUE THREAD

**Issue by riramar** [mtslzr/docker-laravel-boilerplate#1]
```
$ docker exec -ti laravel-app php artisan key:generate

Warning: require(/var/www/vendor/autoload.php): failed to open stream: No such file or directory in /var/www/artisan on line 18

Fatal error: require(): Failed opening required '/var/www/vendor/autoload.php' (include_path='.:/usr/local/lib/php') in /var/www/artisan on line 18
```

**Comment by mtslzr** [comment]
Created #2 to resolve this. Back when I created this, I couldn't find a decent way to incorporate composer into the mix, which causes `php artisan key:generate` to fail.

Added it to the `laravel/` folder and added a command to update it, which should resolve your issue.

Goes without saying, this is mostly a test project for jamming Laravel into Docker. Would probably not recommend a static Composer setup like this for a real project, but nevertheless, thanks for reporting the bug!
